README.md

```diff
@@ -3,7 +3,9 @@ This repository is a set of bash and Python scripts that wrap around rsync.
 
```

Comment:
For some reason README.md has the executable bit set

Reply:
Fixed